spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -486,7 +486,8 @@ def add_electrodes(recording: se.RecordingExtractor, nwbfile=None, metadata: dic
         else:
             nwb_elec_ids = nwbfile.electrodes.id.data[:]
         for metadata_column in metadata['Ecephys']['Electrodes']:
-            if nwbfile.electrodes is None or metadata_column['name'] not in nwbfile.electrodes.colnames:
+            if (nwbfile.electrodes is None or metadata_column['name'] not in nwbfile.electrodes.colnames) \
+                    and metadata_column['name'] != 'group_name':
```

Comment:
@CodyCBakerPhD what is the purpose of this?

Reply:
@bendichter See [this branch](https://github.com/SpikeInterface/spikeextractors/pull/486).

Reply:
@bendichter Long term, I'm thinking of more clever ways of checking against restricted column names for the electrodes table prior to it being created; something like the get_method_signature that  we use over on nwb-conversion-tools.